### PR TITLE
added new feature: ratio_beyond_r_sigma

### PR DESCRIPTION
**Pull request by MaxBenChrist**

### CODE REVIEW

File: tsfresh/feature_extraction/feature_calculators.py [hunk 1]
```diff
@@ -147,6 +147,20 @@ def variance_larger_than_standard_deviation(x):
...
+    """
+    Ratio of values that are more than r*std(x) (so r sigma) away from the mean of x.
+
+    :param x: the time series to calculate the feature of
+    :type x: iterable
+    :return: the value of this feature
+    :return type: bool
+    """
+    x = np.asarray(x)
+    return (sum(x > np.mean(x) + r * np.std(x)) + sum(x < np.mean(x) - r * np.std(x))) * 1.0/len(x)
```
Comment (nils-braun) [inline tsfresh/feature_extraction/feature_calculators.py]: you do not need the `1.0 / len(x)` - just write `/ len(x)`

Comment (nils-braun) [inline tsfresh/feature_extraction/feature_calculators.py]: can this be written much easier using `abs(x - np.mean(x)) > r * np.std(x)`

Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: yes. it also even faster.

Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: changed it

**Pull request merged**